includes/abilities-api/class-wp-abilities-registry.php

```diff
@@ -208,20 +231,20 @@ public function is_registered( string $name ): bool {
 	 *
 	 * Do not use this method directly. Instead, use the `wp_get_ability()` function.
 	 *
-	 * @since 0.1.0
+	 * @since 6.9.0
 	 *
 	 * @see wp_get_ability()
 	 *
 	 * @param string $name The name of the registered ability, with its namespace.
-	 * @return ?\WP_Ability The registered ability instance, or null if it is not registered.
+	 * @return ?WP_Ability The registered ability instance, or null if it is not registered.
 	 */
 	public function get_registered( string $name ): ?WP_Ability {
 		if ( ! $this->is_registered( $name ) ) {
 			_doing_it_wrong(
 				__METHOD__,
 				/* translators: %s: Ability name. */
 				sprintf( esc_html__( 'Ability "%s" not found.' ), esc_attr( $name ) ),
```

Comment:
Not related to this changes but I guess instead of esc_attr( $name ) we should have esc_html( $name ), as name is not used as an html attribute but as a child of an html element.